cvat-core/src/session.ts

```diff
@@ -768,7 +768,12 @@ export class Task extends Session {
     public readonly sourceStorage: Storage;
     public readonly targetStorage: Storage;
     public readonly organization: number | null;
-    public readonly progress: { count: number; completed: number };
+    public readonly progress: {
```

Comment:
Is this change related to this pull request?

Reply:
Yes, partially. We will need progress for Tasks table on Project quality page. But interface provided here is not correct. Actual correct value is [already being used](https://github.com/cvat-ai/cvat/blob/develop/cvat-core/src/session.ts#L852). So this is just type fix